### ISSUE TITLE
Training code for CIFAR

### ISSUE THREAD

**Issue by milliema** [bluecdm/Long-tailed-recognition#3]
Hi, can you please release the training code for CIFAR dataset?
In addition, when changing data from imagenet to inat, error happens as INaturalist_moco only returns [sample1, sample2], label, while ImageNetLT_moco returns 3 items including the index. I wonder for the inat training, do you still use main.py? Or is there other specific code for it? 
Thank you.

**Comment by bluecdm** [comment]
Sorry for the confusion. We added the training code for CIFAR dataset and updated the file 'inat_moco.py'.

**Comment by milliema** [comment]
Thanks for the prompt reply! @bluecdm 
After checking the CIFAR training code, I feel a bit confused about the training settings.
In Imagenet, 2 losses are used to train the student model (gamma-weighted CE + beta-weighted CL);
In CIFAR, besides the 2 losses, an extra distill loss "loss_div" is used with weight alpha=1.0, why? Will the accuracy drop if we set alpha to 0 as in other datasets?

**Comment by bluecdm** [comment]
Yes, setting alpha=0 can result in a performance drop on the CIFAR dataset. We leverage the KL_div loss to alleviate overfitting, which often occurs on CIFAR dataset.
